src/index.js

```diff
@@ -6,7 +6,30 @@ const parseRepo = require("parse-repo");
 const util = require("util");
 
 class NoMoreMastersCommand extends Command {
+  async init() {
+    var defaultBranch = "production"
+    try {
+      defaultBranch = (await execa("git", ["config", "--get", "core.defaultBranch"])).stdout
```

Comment:
Makes sense, but this needs some sort of check in the event that `git config --get core.defaultBranch` is not set. I think `stdout` would return an empty string here.